presto-tests/src/test/java/com/facebook/presto/resourcemanager/TestDistributedClusterStatsResource.java

```diff
@@ -194,4 +201,46 @@ private ClusterStatsResource.ClusterStats getClusterStats(boolean followRedirect
 
         return client.execute(request, createJsonResponseHandler(jsonCodec(ClusterStatsResource.ClusterStats.class)));
     }
+
```

Comment:
Let's reuse the `getClusterStats` instead of copying the logic. The only difference is the param, which we can pass in default false. And for this case we can pass it true.